eogrow/pipelines/export_maps.py

```diff
@@ -102,15 +103,14 @@ def run_procedure(self) -> Tuple[List[str], List[str]]:
         folder = self.storage.get_folder(self.config.output_folder_key)
         crs_eopatch_dict = self.eopatch_manager.split_by_utm(successful)
 
-        # TODO: This could be parallelized per-crs
         for crs, eopatch_list in crs_eopatch_dict.items():
             LOGGER.info("Processing for UTM %d", crs.epsg)
 
             output_folder = fs.path.join(folder, f"UTM_{crs.epsg}")
             # manually make subfolder, otherwise things fail on S3 in later steps
             self.storage.filesystem.makedirs(output_folder, recreate=True)
 
-            merged_map_path = fs.path.join(output_folder, self.get_geotiff_name("full_merged_map"))
+            merged_map_path = fs.path.join(output_folder, self.get_geotiff_name("merged", crs=crs))
```

Comment:
I prefer the constant strings to be defined as class constants. But this is just a stylistic preference, you can leave it as is if you wish.

Reply:
done